ISOLATE_STDLIB_MODULES.rst

```diff
@@ -0,0 +1,240 @@
+PEP: XXX
+Title: Isolating modules in the standard library
+Author: Petr Viktorin <encukou@gmail.com>, Erlend Egeberg Aasland <erlend.aasland@protonmail.com>
+Discussions-To: XXX
+Status: Draft
+Type: Standards
+Requires: 489, 573, 630
+Created: 16-Feb-2022
+Python-Version: 3.11
+Post-History: XXX
+
+
+Abstract
+========
+
+Extensions in the standard library will be converted to multi-phase
+initialization (:pep:`489`) and where possible, all state will be
+stored on module objects rather than in process-global variables.
+
+
+Note on Backdating
+==================
+
+Much of this proposal has already been implemented, partly to solve related
+issues and partly because of misunderstandings and lack of coordination.
+
+We submit this PEP to explain the changes, seek consensus on
+whether they are good,  propose changing modules that do not exhibit
+issues, and set best practices for new modules.
+
+We are aware that the pre-existing changes might need to be reverted if issues
+are found.
+
+The PEP takes the form of the proposal that should have been submitted,
+talking about the situation before the proposed changes as current.
```

Comment:
We should consider removing this sentence or rewording it so it is easier to grasp.

Reply:
Do you have a suggestion?

Reply:
I'll try to come up with something.